app/code/Magento/InventoryAdminUi/Model/Config/Source/Type.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryAdminUi\Model\Config\Source;
+
+use Magento\Framework\Data\OptionSourceInterface;
+use Magento\Inventory\Model\ResourceModel\Source\Type as resourceModel;
+
+/**
+ * Class Type
+ * @package Magento\InventoryAdminUi\Model\Config\Source
+ */
+class Type implements OptionSourceInterface
+{
+    /**
+     * @var \Magento\Cms\Model\Page
+     */
+    protected $resourceModel;
```

Comment:
```suggestion
    private $resourceModel;
```

Best practices suggest using only `private` and `public`. `protected` encourages extending the class, which is not really a best scenario for this class